climpred/tests/conftest.py

```diff
@@ -251,6 +276,30 @@ def hindcast_hist_obs_1d(
     return hindcast
 
 
+@pytest.fixture
+def hindcast_recon_1d_mm(hindcast_recon_1d_ym):
+    hindcast = hindcast_recon_1d_ym.sel(time=slice('1964', '1970'))
+    hindcast._datasets['initialized'].lead.attrs['units'] = 'months'
+    hindcast._datasets['observations']['recon'] = (
+        hindcast._datasets['observations']['recon']
```

Comment:
`hindcast.get_observations()`

Reply:
lets change this once we have easier setters consistently